src/pie-chart/__tests__/pie-chart.test.tsx

```diff
@@ -528,6 +528,29 @@ describe('Details popover', () => {
     expect(detailPopover?.findHeader()?.getElement()).toHaveTextContent(defaultData[0].title);
   });
 
+  test('allow mouse to be move between segment and popover ', () => {
```

Comment:
Nitpick: typo in "to be move" (could be either "to move" or "to be moved" instead)

Reply:
grammar is hard. Since this issue is kinda critical, I am going to merge and fix it next time